dependencies.rosinstall

```diff
@@ -1,5 +1,5 @@
 - git: {local-name: catkin_simple, uri: 'https://github.com/catkin/catkin_simple'} 
 - git: {local-name: eigen_catkin, uri: 'https://github.com/ethz-asl/eigen_catkin'}
 - git: {local-name: grid_map, uri: 'https://github.com/Jaeyoung-Lim/grid_map.git', version: c45c300ce3930312dcb4deadc811ccb4eaeb4bd0}
-- git: {local-name: grid_map_geo, uri: 'https://github.com/ethz-asl/grid_map_geo.git'}
+- git: {local-name: grid_map_geo, uri: 'https://github.com/ethz-asl/grid_map_geo.git', version: 64161b8d30b083f362052411e2ca0d6f494bfac3}
```

Comment:
You could also pin it to branch: main

Reply:
Sure, the branch in `grid_map_geo` is called master, will pin it there. While I'm at it, shall I do the same for `grid_map` too? When I try to look up the commit that we're currently pinning to https://github.com/ethz-asl/grid_map/commit/c45c300ce3930312dcb4deadc811ccb4eaeb4bd0, there's a warning it's either on a fork or on a branch that doesn't exist anymore.

Reply:
Yes, pin all the branches.

Reply:
@Ryanf55  @Jaeyoung-Lim done, for all dependencies. I've checked that the image builds, do I need to do anything else?

Reply:
LGTM. Good job

Reply:
Thanks! :robot: